tests/travis/install-mssql-pdo_sqlsrv.sh

```diff
@@ -3,4 +3,9 @@
 set -ex
 
 echo "Installing extension"
-pecl install pdo_sqlsrv
+
+if [ "$TRAVIS_PHP_VERSION" == "7.3" ]; then
+  pecl install pdo_sqlsrv-devel
```

Comment:
Can we pin it to `5.4.0-preview`? If `devel` gets broken after `5.4.0` is released but before we update this config, I don't want our builds to fail for no reason.